.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["es2015", "react"]
```

Comment:
Instead of using `es2015`, you may want to use [`babel-preset-env`](https://github.com/babel/babel-preset-env) instead. It's more future-proof and includes `es2015`. It's pretty cool!